skills/declarative/calendarSkill/runtime/customaction/Actions/MSGraph/CheckAvailability.cs

```diff
@@ -19,9 +19,9 @@ namespace Microsoft.BotFramework.Composer.CustomAction.Actions.MSGraph
     /// This has proven to be more consistent that LUIS datetime recognition by allowing the use of the 
     /// user's current timezone time as a relative datetime rather than a mix of absolute times and UTC times.
     /// </summary>
-    [MsGraphCustomActionRegistration(CheckAvailability.DeclarativeType)]
     public class CheckAvailability : Dialog
     {
+        [JsonProperty("$kind")]
         public const string DeclarativeType = "Microsoft.Graph.Calendar.CheckAvailability";
```

Comment:
This isn't using the same => syntax as all the others

Reply:
This is deliberately keeping the same as the old existing style. The new style is basically kept for MsGraph custom actions and this isn't an MsGraph custom action, it has more to do with transforming the result from graph and filtering things. Eventually when I break out the MsGraph custom action package this would stay with the calendar skill.